.github/workflows/publish.yml

```
@@ -6,46 +6,52 @@ on:
       source:
         required: false
         type: string
+        default: https://api.nuget.org/v3/index.json
+        description: The NuGet server URL used by the `dotnet nuget push` command's `--source` argument. 
       verbosity:
         required: false
         type: string
         default: minimal
+        description: The logging verbosity type used by the `dotnet` command.
     secrets:
       # We can't access org secrets here so they need to be passed in, see:
       # https://github.community/t/resuable-called-workflow-environment-variables-secrets-and-trigger-event-access/207723/2
       API_KEY:
         required: true
 
+defaults:
+  run:
+    shell: pwsh
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      # v1.1.0
-      - uses: haya14busa/action-cond@c69871f4d4693e93126e770d7c3588cb1d473462
-        id: source-value
-        with:
-          cond: ${{ inputs.source == '' }}
-          if_true: "https://api.nuget.org/v3/index.json"
-          if_false: "${{ inputs.source }}"
+      - name: Print Source
+        shell: pwsh
+        run: |
+          $source = @'
+          ${{ inputs.source }}
+          '@.Trim()
 
-      - name: Print source
-        run: echo NuGet Source:${{ steps.source-value.outputs.value }}
+          Write-Output "NuGet Source: $source"
+          "SOURCE_URL=$source" >> $Env:GITHUB_ENV
 
-      - name: Get the version
-        id: get-version
+      - name: Get the Version
         run: |
-          VERSION="${GITHUB_REF_NAME//v}"
-          echo VERSION:${VERSION}
-          echo ::set-output name=VERSION::${VERSION}
+          $version = $Env:GITHUB_REF_NAME.Trim().Substring(1)
```

The following would convey the intent better:
```suggestion
          $version = $Env:GITHUB_REF_NAME.Trim().TrimStart("v")
```